pyproject.toml

```diff
@@ -7,8 +7,15 @@ authors = ["Cristobal Contreras <crisconru@gmail.com>, Jose Rivera <josemrivera@
 [tool.poetry.dependencies]
 python = "^3.8"
 fastapi = "^0.63.0"
+uvicorn = "^0.13.4"
 
 [tool.poetry.dev-dependencies]
+flake8 = "^3.8.4"
+mypy = "^0.812"
+black = "^20.8b1"
+
+[tool.poetry.scripts]
+serve = "uvicorn app.main:app --reload"
```

Comment:
Oh I think I know why this fails, brings me back memories of Flask + gunicorn. I'll raise a quick PR after merging this.